### PR TITLE
Updated some stuff.

### PR DESCRIPTION
**Pull request by lambdagg**
refer to the commits for more insights.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -31,24 +51,23 @@ public class Program {
...
+host machine.
 
+one way of getting around this problem would be to extract the library from the
+jar archive to a known folder, then to load it. this approach works but also
+requires extracting and loading the correct library for the wanted processor
+architecture (arch) and operating system (OS).
 
-[yanl](https://github.com/stardust-enterprises/yanl) takes care of platform
-detection, library extraction and location.
+and that is where [yanl][yanl] performs: we take care of platform detection,
```
Comment (qtdyn) [inline README.MD]: and "this" is where

File: README.MD [hunk 2]
```diff
@@ -31,24 +51,23 @@ public class Program {
...
-
-one way of getting around this problem would be to extract the library from
-the jar archive to a known folder, then to load it. this approach works
-but also requires extracting and loading the correct library for your
-processor architecture and operating system. 
+host machine.
 
+one way of getting around this problem would be to extract the library from the
+jar archive to a known folder, then to load it. this approach works but also
+requires extracting and loading the correct library for the wanted processor
```
Comment (qtdyn) [inline README.MD]: for the "correct"

File: README.MD [hunk 3]
```diff
@@ -1,27 +1,47 @@
...
-[JVM](https://adoptium.net), written in [Kotlin](https://kotlinlang.org).
+# yanl - yet another native loader [![Maven Central][badge-mvnc]][yanl-mvnc]
+yet another native library loader and extractor for the [JVM][jvm], written in
+[Kotlin][kotlin].
 
 # why
-other libraries simply don't fit my needs and/or aren't as 
-expandable/configurable as I'd like.
+other libraries simply don't fit my needs and/or are not as
+expandable/configurable as we'd like.
```
Comment (qtdyn) [inline README.MD]: as "I'd" pas "we'd" pd ya le singulier juste au dessus reste consistant

File: README.MD (hunk 1 above)
Comment (lambdagg) [reply README.MD]: f4482aa

File: README.MD (hunk 3 above)
Comment (lambdagg) [reply README.MD]: f4482aa

File: README.MD (hunk 2 above)
Comment (lambdagg) [reply README.MD]: f4482aa

File: README.MD [hunk 4]
```diff
@@ -1,49 +1,40 @@
...
-yet another native library loader and extractor for the [JVM][jvm], written in
-[Kotlin][kotlin].
+
+yet another native library loader and extractor for the [JVM][jvm], written in [Kotlin][kotlin].
 
 # why
-other libraries simply don't fit my needs and/or are not as
-expandable/configurable as we'd like.
+
+other libraries simply don't fit my needs and/or are not as expandable/configurable as we would like.
```
Comment (qtdyn) [inline README.MD]: """""we""""" >:(

Comment (qtdyn) [reply README.MD]: meow

Comment (lambdagg) [reply README.MD]: 02dfb28

**Comment by qtdyn** [comment]
Gaming

**Pull request merged**